commands/stats.js

```diff
@@ -88,22 +88,67 @@ const stats = msg => {
 
 	const getCompUserStats = (error, response, parsedBody) => {
 		const competitiveData = parsedBody.us.stats.competitive;
+		const compGmStats = competitiveData.game_stats;
+		const compOvrStats = competitiveData.overall_stats;
 		const emptyObjectCompChecker = Object.getOwnPropertyNames(competitiveData);
 		if (emptyObjectCompChecker.length === 0 || emptyObjectCompChecker.length === 0) {
 			msg.reply("Sorry. This user has missing data. Boo boo doo de doo.");
 			return;
 		}
-		// Sucrizzle - Playing with Git commits
+
+		// Make sure the user has played some games this comp Season.  Not sure if it would be null or 0 or ???  Taking a guess here
+		if (compOvrStats.games == null || compOvrStats.games == 0) {
```

Comment:
In overall_stats it comes back as:
"games": 36

In game_stats it comes back as:
"games_played": 36.0

If we fork OWAPI we could standardize the terms